src/main/java/org/apache/cassandra/distributed/api/ICluster.java

```diff
@@ -54,6 +56,15 @@
 
     IMessageFilters filters();
 
+    /**
+     * dynamically sets the current uncaught exceptions filter
+     *
+     * the predicate should return true if we should ignore the given throwable on the given instance
+     *
+     *
+     */
+    void setUncaughtExceptionsFilter(BiPredicate<Integer, Throwable> ignoreThrowable);
+
```

Comment:
would be nice to have this overload as well

```
default void setUncaughtExceptionsFilter(Predicate<Throwable> ignoreThrowable)
{
    setUncaughtExceptionsFilter((ignore, error) -> ignoreThrowable.test(error));
}
```

some cases may not care about the instance and just want to ignore the error on all nodes, so this overload would be nice short hand for that.